lib/computer_player.rb

```diff
@@ -1,11 +1,23 @@
-class ComputerPlayer
+class Computer_player
```

Comment:
Ruby naming conventions - `snake_case` for methods, `CamelCase` for classes.
https://github.com/rubocop-hq/ruby-style-guide#camelcase-for-classes